addons/structuredtypes/plugins/fr.gouv.mindef.safran.is.design/src/fr/gouv/mindef/safran/is/propertyTesters/ReferencePropertyTester.java

```diff
@@ -0,0 +1,44 @@
+package fr.gouv.mindef.safran.is.propertyTesters;
+
+import org.eclipse.core.expressions.PropertyTester;
+import org.eclipse.sirius.viewpoint.DSemanticDecorator;
+import org.obeonetwork.dsl.environment.MultiplicityKind;
+import org.obeonetwork.dsl.environment.Reference;
+
+/**
+ * {@link PropertyTester} for asserting that a {@link Reference} is bidirectionnal with two multiplicities.
+ * 
+ * @author Thibault Béziers la Fosse <a href="mailto:thibault.beziers-la-fosse@obeo.fr">thibault.beziers-la-fosse@obeo.fr</a>
+ *
+ */
+public class ReferencePropertyTester extends PropertyTester {
+
+	private static final String IS_NN = "isNN";
```

Comment:
why not calling it isMultiple or isMany as in ECore ?

Reply:
Good call. I'll update it